cs/src/comm/epoxy-transport/Frame.cs

```diff
@@ -48,10 +50,12 @@ public Framelet(FrameletType type, ArraySegment<byte> contents)
 
         public static bool IsKnownType(UInt16 value)
         {
-            return value == (UInt16)FrameletType.LayerData
-                | value == (UInt16)FrameletType.PayloadData
-                | value == (UInt16)FrameletType.EpoxyHeaders
+            return false
```

Comment:
Yeah, I don't think it's worth doing anything even slightly clever here. The false causes an unnecessary pause while reading it and the single pipes make me wonder what bit twiddling you're up to. return value == a || value == b?